extensions-core/druid-ranger-security/pom.xml

```diff
@@ -34,6 +34,10 @@
         <relativePath>../../pom.xml</relativePath>
     </parent>
 
+    <properties>
```

Comment:
You should change that in the master pom.xml of Druid, not here, it should be removed from here